v2ray/vmess/outbound.go

```diff
@@ -0,0 +1,188 @@
+package vmess
+
+// the code below is mostly ripped from sager-box project under GPL
+// Copyright (C) 2022 by nekohasekai <contact-sagernet@sekai.icu>
+// at the moment only used for testing
```

Comment:
Since this is only used for testing, I would rename the file to `outbound_test.go`. Even though it doesn't actually contain any tests itself, it'll be very obvious that's all it's used for.